datasets/gov/vat/r27_aprob_sekl_pasel.csv

```diff
@@ -0,0 +1,12 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vat/aprob_sekl_pasel,,,,,,,,,,,,,
```

Comment:
Kodėl nerašote pilnais žodžiai `(...)/aprobuoti_sekliniai_paseliai` ar pan.?

Reply:
pastebėjau, jog kolegos trumpina pavadinimus kituose rinkiniuose. Šiaip trumpinu dėl to, jog pristingu pavadinimų dataset ir model laukuose, tai kad jie tiesiogiai nesidubliuotų (kitu atveju vieną iš jų tenka keisti), vieną jų tiesiog sutrumpinu.